capabilities/security.html

```diff
@@ -0,0 +1,31 @@
+---
+layout: capability
+
+asset-name: security
+
+description: "Security is the most important defense against people who seek to harm
+your business. See what we can offer. This is one of our strengths!"
+
+tags: [Security]
+
+title: Security
+
+subtitle: "Absolutely the most important element of a software."
+
+text: '<p>We have a lot of experience specifically with Single Sign On.</p>
+<p>Google, Facebook, GitHub, and Twitter do this – we do too. and many more . It’s the
```

Comment:
This sentence has punctuation and spacing errors:
Google, Facebook, GitHub, and Twitter do this – we do too. and many more .

Suggested fix:
Google, Facebook, GitHub, Twitter and many more do this; we do too.

Reply:
Yes

Reply:
Fixed.